test/cpp/interop/xds_interop_client.cc

```diff
@@ -332,6 +347,15 @@ class XdsUpdateClientConfigureServiceImpl
       if (metadata_iter != metadata_map.end()) {
         config.metadata = metadata_iter->second;
       }
+      if (absl::GetFlag(FLAGS_request_payload_size) > 0) {
+        config.request_payload_size = absl::GetFlag(FLAGS_request_payload_size);
+        std::string payload(config.request_payload_size, '0');
+        config.request_payload = payload;
```

Comment:
Please print a warning log message when `request_payload_size` > 0, but an `EmptyCall` is requested.

Reply:
Done